pom.xml

```diff
@@ -31,7 +31,7 @@
 
   <groupId>org.codehaus.mojo</groupId>
   <artifactId>properties-maven-plugin</artifactId>
-  <version>1.0.1-SNAPSHOT</version>
+  <version>1.0.1-pin1</version>
```

Comment:
Oh, right, sorry. This is my internal version. Will restore SNAPSHOT tomorrow.

Gesendet von Yahoo Mail auf Android 
 
  Am Do., 30. Juni 2016 um 20:14 hat Karl Heinz Marbaise<notifications@github.com> Folgendes geschrieben:   
In pom.xml:
> @@ -31,7 +31,7 @@
>  
>    <groupId>org.codehaus.mojo</groupId>
>    <artifactId>properties-maven-plugin</artifactId>
> -  <version>1.0.1-SNAPSHOT</version>
> +  <version>1.0.1-pin1</version>

Why would you like to change the version? Keep the SNAPSHOT

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub, or mute the thread.